content/en/security/application_security/threats/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Application Threat Monitoring and Protection
+kind: documentation
+further_reading:
+- link: "/security/application_security/threats/add-user-info/"
+  tag: "Documentation"
+  text: "Tracking User Activity"
+- link: "/security/application_security/threats/setup_and_configure/"
+  tag: "Documentation"
+  text: "Configuring your ASM setup"
+- link: "/security/application_security/risk_management/"
+  tag: "Documentation"
+  text: "Risk Management"
+- link: "/security/application_security/how-appsec-works/"
+  tag: "Documentation"
+  text: "How ASM Works"
+---
+
+ASM Threat Monitoring and Protection uses trace telemetry from your APM-instrumented applications to identify threats and attacks on your running services by comparing the observed behavior against known attack patterns. 
+
+Security signals raised by Threat Monitoring are summarize and surfaced in views you already commonly visit to monitor service health and performance. The [Service Catalog][1] and individual Service Pages in APM provide quick insights into application threat signals, and let you quickly click through to investigate signals and block attackers.
+
+{{< img src="security/application_security/threats-on-svc-cat.png" alt="Service Catalog with services showing threat signals" style="width:100%;" >}}
+
+For additional information about how threat monitoring and protection works, read [How ASM Works][4].
+
+
+## Explore threat signals
+
+When threat data is coming in for your services, ASM Overview shows a summary of what's happening. Here, you can review security monitoring coverage, and enable ASM on services. To investigate signals of suspicious activity, click a service's **Review** link.
```

Comment:
Should "coming in for your services" be "coming for your services" or "coming into your services"?

Reply:
```suggestion
When threat data for your services is coming into Datadog, ASM Overview shows a summary of what's happening. Here, you can review security monitoring coverage, and enable ASM on services. To investigate signals of suspicious activity, click a service's **Review** link.
```